tests/execution/add-field-modify.md

```diff
@@ -43,6 +43,8 @@ type Query {
 ```yml @test
 - method: POST
   url: http://localhost:8080/graphql
+  headers:
+    Accept: application/graphql-response+json
```

Comment:
Move this logic into execution spec so that by default we make the request with headers.

Reply:
I accept these are changes in many tests files.
Realistically, headers comes form request which we define in tests files. That should be the source of truth.
Also it will be the problem for us to write tests for non compliance.